Makefile

```diff
@@ -1,9 +1,8 @@
 VERSION := $(shell cat VERSION)
-PY2VERSION := $(shell cat PY2VERSION)
 
 define build-image
```

Comment:
Can we move this into the build step? I only defined it so it could be paramaterised for py2 and 3 builds.